composer.json

```diff
@@ -14,6 +14,9 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zend-stdlib": "2.*"
+        "laminas/laminas-stdlib": "3.2.1"
```

Comment:
is the new version fully compatible?

Reply:
Looks like yes. At least no failed tests and every new tag since 2.7 either a bugfix or new feature or PHP version support